admob/src/android/banner_view_internal_android.h

```diff
@@ -48,6 +48,7 @@ namespace internal {
 
 class BannerViewInternalAndroid : public BannerViewInternal {
  public:
+  // The following line should trigger a lint warning.
   BannerViewInternalAndroid(BannerView* base);
```

Comment:
⚠️ Lint warning: `Single-parameter constructors should be marked explicit.`<hidden value="cpplint-file-comment"></hidden><hidden value="admob/src/android/banner_view_internal_android.h:52:  Single-parameter constructors should be marked explicit.  [runtime/explicit] [5]"></hidden>